### PR TITLE
appveyor: enable werror for the mscng backend

### PR DESCRIPTION
**Pull request by vmiklos**
This works for me locally, so start with that as a first step.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -38,34 +38,41 @@ environment:
...
 
     - COMPILER: MinGW-w64
       STATIC: no
       UNICODE: yes
+      WERROR: no
     
     - COMPILER: MinGW-w64
       STATIC: yes
       UNICODE: no
+      WERROR: no
```
Comment (lsh123) [inline appveyor.yml]: I don't think you are using this flag for MinGW builds

Comment (vmiklos) [reply appveyor.yml]: Indeed, also at the other place.

Comment (lsh123) [reply appveyor.yml]: Was not sure if you missed something or it's intentional.

Comment (vmiklos) [reply appveyor.yml]: The previous. :-)

**Pull request merged**